modules/indexer/code/bleve/bleve.go

```diff
@@ -284,6 +284,8 @@ func (b *Indexer) Search(ctx context.Context, opts *internal.SearchOptions) (int
 		searchRequest.AddFacet("languages", bleve.NewFacetRequest("Language", 10))
 	}
 
+	searchRequest.SortBy([]string{"-_score", "RepoID"})
```

Comment:
Maybe the second sort factor should be updated time but not repo_id?

Reply:
Cool. I think it makes total sense.

I'm gonna change it use the update time instead of the repo id.

Reply:
@lunny Done